pkg/exporter/sumologicexporter/README.md

```diff
@@ -4,37 +4,49 @@ This exporter supports sending logs and metrics data to [Sumo Logic](https://www
 
 The following configuration options are supported:
 
-- `endpoint` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
-- `compress_encoding` (optional): Compression encoding format, either empty string (`""`), `gzip` or `deflate` (default `gzip`).
-Empty string means no compression
-- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default `1_048_576` (1MB) is used.
+- `endpoint` (required, unless configured with [`sumologicextension`][sumologicextension]): Unique URL generated for your HTTP Source.
+  This is the address to send metrics to.
+- `compress_encoding` (optional): Compression encoding format. Empty string means no compression
+  - default: `gzip`
+  - allowed values: `gzip`, `deflate`, `""`
```

Comment:
IMHO this indicates, that `default` and `allowed_values` is subconfiguration for `compress_encoding`.(see `queue_size` for example). Could we mark `default` and `allowed values` without bullet points?

Reply:
Yeah... I was thinking about that but then when it's all in one line it's all a huge pile of information and it's hard to find a particular setting/property that you're looking for (e.g. if you want to find the default value you have to peruse the whole bullet point entry to find it :/ )

Reply:
We could put as separate line in this bullet point

list:
  - `compress_encoding` (optional): Compression encoding format. Empty string means no compression
    default: `gzip`
    allowed values: `gzip`, `deflate`, `""`

```
list:
  - `compress_encoding` (optional): Compression encoding format. Empty string means no compression
    default: `gzip`
    allowed values: `gzip`, `deflate`, `""`
```